packages/schematics/angular/universal/index_spec.ts

```diff
@@ -150,10 +150,12 @@ describe('Universal Schematic', () => {
     const tree = schematicRunner.runSchematic('universal', defaultOptions, appTree);
     const filePath = '/projects/bar/src/main.ts';
     const contents = tree.readContent(filePath);
-    expect(contents).toMatch(/document.addEventListener\('DOMContentLoaded', \(\) => {/);
+    // tslint:disable:max-line-length
```

Comment:
This would do as well.

Small nit. This can be simplied and won't require to disable tslint
```
    expect(contents)
      .toMatch(/document.addEventListener\('DOMContentLoaded', \(\) => {[\w\W]+;[\r\n]}\);/);
```

Reply:
Done with commit 73c22d7